libsupport/CMakeLists.txt

```diff
@@ -0,0 +1,40 @@
+add_library(galois_support STATIC)
+add_library(Galois::support ALIAS galois_support)
+set_target_properties(galois_support PROPERTIES EXPORT_NAME support)
+add_dependencies(lib galois_support)
+
+set(sources
+        src/GetEnv.cpp
+        src/Logging.cpp
+)
+
+target_sources(galois_support PRIVATE ${sources})
+
+target_include_directories(galois_support PUBLIC
```

Comment:
Given that we only support fmt 6.0 and later, we should check for that.

Reply:
I actually back ported support to fmt 4.0 to support the standard fmt package in bionic. Let me update the docs and the CMake `find_packages`.

Reply:
Sounds good. Thanks!